src/__tests__/utils.js

```diff
@@ -21,6 +21,10 @@ const countESLintError = (eslintFilePath, filePattern, testType) => {
     { shell: true, encoding: 'utf-8' },
   );
 
+  if (!spawn.stdout.length) {
+    throw spawn.stderr;
```

Comment:
We want to throw some readable error, in case the eslint config is invalid, instead of having `Unexpected end of JSON input`:

![Screenshot 2023-10-16 at 09 26 23](https://github.com/smg-automotive/eslint-config-pkg/assets/2486360/23557742-0aa7-40e2-966c-59332398c5b8)